css/style.css

```diff
@@ -0,0 +1,195 @@
+body {
+  /* font-size: 40px; */
```

Comment:
Try to not leave commented code in the code you're submitting for review. Reason is - this code will probably live there forever and won't be removed (this is what happening in real life projects)

Reply:
Yes, sorry I forgot to delete it. Done. Thanks